src/overflow_check_conditional.rs

```diff
@@ -0,0 +1,41 @@
+use rustc::lint::*;
+use rustc_front::hir::*;
+use utils::{span_lint, span_help_and_lint, snippet, span_lint_and_then};
+
+/// **What it does:** This lint finds classic overflow checks.
+///
+/// **Why is this bad?** Most classic C overflow checks will fail in Rust. Users can use functions like overflowing_* and wrapping_* instead.
```

Comment:
Can you put `overflowing_*` and `wrapping_*` between quotes? Looks better on the wiki.
```markdown
`overflowing_*` and `wrapping_*`
```